migration/incremental/033-metric-rollup.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE _prom_catalog.rollup (
+    id             SERIAL,
+    name           TEXT NOT NULL PRIMARY KEY,
+    schema_name    TEXT NOT NULL,
+    resolution     INTERVAL NOT NULL,
+    retention      INTERVAL NOT NULL
+);
+GRANT SELECT ON TABLE _prom_catalog.rollup TO prom_reader;
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE _prom_catalog.rollup TO prom_writer;
+
+CREATE TABLE _prom_catalog.metric_rollup (
+    id              SERIAL,
```

Comment:
The id should be the primary key, and the (rollup_id, metric_id) should have a unique constraint.